src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/integrations/IntegrationPeripheralProvider.java

```diff
@@ -1,27 +1,20 @@
 package de.srendi.advancedperipherals.common.addons.computercraft.integrations;
 
 import dan200.computercraft.api.ComputerCraftAPI;
-import dan200.computercraft.api.peripheral.IPeripheral;
-import dan200.computercraft.api.peripheral.IPeripheralProvider;
 import de.srendi.advancedperipherals.AdvancedPeripherals;
 import de.srendi.advancedperipherals.common.util.Platform;
 import de.srendi.advancedperipherals.lib.integrations.IPeripheralIntegration;
 import de.srendi.advancedperipherals.lib.peripherals.BlockEntityIntegrationPeripheral;
-import net.minecraft.core.BlockPos;
-import net.minecraft.core.Direction;
-import net.minecraft.world.level.Level;
 import net.minecraft.world.level.block.NoteBlock;
 import net.minecraft.world.level.block.entity.BlockEntity;
-import net.minecraftforge.common.util.LazyOptional;
-import org.jetbrains.annotations.NotNull;
 
 import java.util.Comparator;
 import java.util.Optional;
 import java.util.PriorityQueue;
 import java.util.function.Function;
 import java.util.function.Predicate;
 
-public class IntegrationPeripheralProvider implements IPeripheralProvider {
+public class IntegrationPeripheralProvider {
 
     private static final String[] SUPPORTED_MODS = new String[]{"powah", "create", "mekanism", "botania"};
```

Comment:
Propably. But the integrations are still broken, since I am not sure what the new alternative for peripheral providers are in CC. I also didn't try to fix this yet since we don't have any addon mod that supports 1.20.4

Well except for the two mekanism radiation functions